subscriptions/schema.py

```diff
@@ -0,0 +1,77 @@
+import graphene
+from django.db import transaction
+from graphene import relay
+from graphene_django.filter import DjangoFilterConnectionField
+from graphene_django.types import DjangoObjectType
+from graphql_jwt.decorators import login_required
+
+from .models import Subscription, SubscriptionType, SubscriptionTypeCategory
+
+
+class SubscriptionNode(DjangoObjectType):
+    class Meta:
+        model = Subscription
+        interfaces = (relay.Node,)
+        filter_fields = []
+
+
+class SubscriptionTypeNode(DjangoObjectType):
+    label = graphene.String()
+
+    class Meta:
+        model = SubscriptionType
+        interfaces = (relay.Node,)
+        filter_fields = []
+        exclude = ("subscriptions",)
+
+
+class SubscriptionTypeCategoryNode(DjangoObjectType):
+    label = graphene.String()
+
+    class Meta:
+        model = SubscriptionTypeCategory
+        interfaces = (relay.Node,)
+        filter_fields = []
+
+
+class SubscriptionInputType(graphene.InputObjectType):
+    subscription_type_id = graphene.ID(require=True)
+    enabled = graphene.Boolean(require=True)
```

Comment:
Hmm, not sure why this isn't failing somehow, but isn't that supposed to be `required` ? 🙂

Reply:
Good catch. I have no idea why I had that same typo in 2 different files 😑 👍